ground/gcs/src/plugins/coreplugin/mainwindow.cpp

```diff
@@ -344,10 +344,17 @@ void MainWindow::extensionsInitialized()
         }
         if(showDialog)
         {
+            // This has often ended up behind the splash screen, which looks
```

Comment:
This is the actual change.  The rest is just pumping signals to slots up the way.